spec/models/miq_expression/miq_expression_spec.rb

```diff
@@ -590,9 +590,42 @@
     expect(filter.to_ruby).to eq('<value ref=vm, type=integer>/virtual/used_disk_storage</value> >= 1048576000')
   end
 
+  it 'should test .is_numeric' do
```

Comment:
Better :+1:. I think it could be still better further separated as something like:

```ruby
describe ".is_numeric?" do
  it "returns true for a string of comma-separated digits" do
   expect(described_class.is_numeric?("10,000,55")).to be_truthy
  end
  # etc....
end
```